debezium-api/src/main/java/io/debezium/config/ConfigurationDefinition.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.config;
+
+public interface ConfigurationDefinition {
+
+    String TOPIC_PREFIX_PROPERTY_NAME = "topic.prefix";
+    String DATABASE_CONFIG_PREFIX = "database.";
```

Comment:
`CommonConnectorConfig.DATABASE_CONFIG_PREFIX` should point here then to avoid duplication?

Reply:
@mfvitale I believe @rk3rn3r wants you to do something like CommonConnectorConfig.DATABASE_CONFIG_PREFIX = ConfigurationDefinition.DATABASE_CONFIG_PREFIX